scripts/hf_sim.py

```diff
@@ -548,7 +562,11 @@ def validate_end(idx_n):
         validate_end(work_idx[-1] + 1)
 
     os.remove(in_stats)
-    print("Process %03d of %03d finished (%.2fs)." % (rank, size, MPI.Wtime() - t0))
+    print(
+        "Process {} of {} completed {} stations ({:.2f}).".format(
```

Comment:
If you are changing it, why not a f-string? Just curious?

Reply:
Just copied from the log below